### PR TITLE
Fix double notification on Android

### PR DESCRIPTION
**Pull request by neven-s**
## Description

**Ticket: [CP-9567]** 

This fixes double notification on Android devices when app is in **background** since FirebaseSDK automatically handles FCM with `notification` payload.

## Testing
- make sure that notifications are enabled in system and app settings
- make sure you are receiving any notifications and hear sound 
- use wallet on another device to send some funds on testing device (can be same wallet)
- observe that no double sound is heard or double notifications present

## Checklist

Please check all that apply (if applicable)
- [ ] I have performed a self-review of my code
- [ ] I have verified the code works
- [ ] I have added/updated necessary unit tests 
- [ ] I have updated the documentation

[CP-9567]: https://ava-labs.atlassian.net/browse/CP-9567?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/core-mobile/app/services/notifications/NotificationsService.ts [hunk 1]
```diff
@@ -321,16 +321,14 @@ class NotificationsService {
         pressAction: {
           id: PressActionId.OPEN_PORTFOLIO,
           launchActivity: LAUNCH_ACTIVITY
-        }
+        },
+        channelId: channelId ?? ''
```
Comment (onghwan) [inline packages/core-mobile/app/services/notifications/NotificationsService.ts:325]: Is there a difference between passing an empty string and undefined for channelId?

Comment (neven-s) [reply packages/core-mobile/app/services/notifications/NotificationsService.ts:325]: yes, in my experience `notifee` will report such payload as invalid

**Review by onghwan — APPROVED** [review]

File: packages/core-mobile/app/services/notifications/NotificationsService.ts (hunk 1 above)
Comment (onghwan) [reply packages/core-mobile/app/services/notifications/NotificationsService.ts:325]: it's weird since `channelId` is an optional parameter.

Comment (neven-s) [reply packages/core-mobile/app/services/notifications/NotificationsService.ts:325]: it's optional for ios platform, on android it's required

**Pull request merged**